portfolio/src/main/webapp/comment-page/comment.js

```diff
@@ -80,7 +80,12 @@ function deleteComments() {
 
 /** Print number of comments chosen by user, or 5 by default. */
 function limitedComments() {
-  fetch('/data')
+  let langCode = document.getElementById('language').value;
+  console.log(langCode);
+
+  const url = '/data?language=' + langCode;
```

Comment:
I really like this logic, taking advantage of the URL to be able change the language!

Reply:
Thanks!